jscomps/EmbedLiteConsoleListener.js

```diff
@@ -202,14 +182,9 @@ $EmbedLiteConsoleListener.prototype = {
       case "embedui:logger": {
         var data = JSON.parse(aData);
         if (data.enabled) {
```

Comment:
Related to the comment https://github.com/sailfishos/embedlite-components/pull/19#discussion_r699142353 if browser app sends that it wants to listen console log when we should respect that regardless of the env variables.